services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2731,6 +2747,20 @@ public SchemaEntry getValueSchema(String clusterName, String storeName, int id)
     return getVeniceHelixAdmin().getValueSchema(clusterName, storeName, id);
   }
 
+  private void validateValueRecordSchema(Schema valueRecordSchema) {
```

Comment:
Can you rename it to be `makeSureRecordSchemaTopLevelFieldsMustHaveDefault`? And potentially we should move this function to `AvroSchemaUtils`.

Reply:
We are dropping this check as of now.